config/purposes/bioscan.yml

```diff
@@ -83,6 +85,7 @@ LBSN-384 PCR 2 Pool:
   :target: StockMultiplexedLibraryTube
   :type: IlluminaHtp::StockTubePurpose
   :creator_class: LabwareCreators::PooledTubesFromWholePlates
+  :transfer_template: '384 plate to tube'
```

Comment:
What does this transfer_template parameter do? Curious as it isn't on the epic branch yet the pooling seems to work. Are you just being more explicit?

Reply:
Is a reference to the transfer template in SS. it is defined there: <https://github.com/sanger/sequencescape/pull/3780/files#diff-36a74a283e00b28f937853568ea928e403aed96cd63b5c9b8d7b05f5632108cb>